### ISSUE TITLE
Discriminate between asymptomatic and mild cases 

### ISSUE THREAD

**Issue by giovannic** [mrc-ide/hypatia#23]
Todo:

- [ ] Integrate parameters for prob_asymp
- [ ] Create IAsymp state
- [ ] Modify infection flow to allow for asymptomatic

```
# new flow
if prob_hosp trips: schedule severe infection
else if prob_asymp trips: schedule asymptomatic
else schedule mild infection
```

**Comment by cwhittaker1000** [comment]
tl;dr This will require creation of a new state, `IAsymp`, which refers to Asymptomatic infections (i.e. those not displaying symptoms), and also a new set of age-specific probabilities `prob_asymp` (specifying the age-specific probability of being asymptomatic conditional on infection). These will be extracted shortly but in the meantime, please use  `prob_asymp <- c(0.3, 0.3, rep(0.2, 15)`.

We need to divide up the Mild disease state (who do not require hospitalisation) we have currently into two states - `IMild` and `IAsymp`. 

I.e. instead of just having `IMild = individual::State$new("IMild", IMild)` in https://github.com/mrc-ide/hypatia/blob/dev/oj_refactor/R/individuals.R#L33, we'll have an additional line of `IAsymp = individual::State$new("IAsymp", IAsymp)`. 

Individuals in `IAsymp` will follow similar transitions as `IMild` individuals - i.e. the only movement from this state will be a Recovery taking them into the `R` state.

This will require specifying a new age-specific probability, `prob_asymp`, which is the probability of being asymptomatic conditional on infection. Estimates of these parameters can be drawn from https://www.medrxiv.org/content/10.1101/2020.05.10.20097543v3, https://onlinelibrary.wiley.com/doi/10.1002/jmv.26326 and possibly https://www.nature.com/articles/s41591-020-0962-9. 

I will work on extracting these and getting literature-derived estimates into the framework, but in the meantime, please use `prob_asymp <- c(0.3, 0.3, rep(0.2, 15)` for any infrastructure development and testing purposes. 

We might also think about having reduced infectivity at some point, but I think we can hold on that for now.

**Comment by ceeb950** [comment]
Closes #23.
